package.json

```diff
@@ -117,7 +117,7 @@
     "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-unused-imports": "^1.1.0",
-    "faker": "^5.3.1",
+    "faker": "5.5.3",
```

Comment:
@jekabs-karklins maybe it is worth putting a comment somewhere about this just in case we don't forget in the future.

Reply:
Looks like there's no way to add comments here (JSON doesn't have a way of adding comments, and there doesn't seem to be a special syntax for `package.json` files). Do you have any suggestions where a comment could go? The [issue](https://github.com/UserOfficeProject/stfc-user-office-project/issues/312) will remain in Github after being closed, but I know people may not notice it.

Reply:
Yes I see. Maybe it is fine to leave it as it is for now. If the version is fixed usually someone should check the reason before upgrading.